### DISCUSSION TITLE
SYNTH_STRATEGY usage

### DISCUSSION THREAD

**Discussion by vijayank88** [The-OpenROAD-Project/OpenLane#959] [Q&A] [answered]
Hi,
@donn  @maliberty FYI

Is anyone have brief details on index usage of DELAY and AREA for synthesis.
```
Strategies for abc logic synthesis and technology mapping
Possible values are DELAY/AREA 0-4/0-3; the first part refers to the optimization target of the synthesis strategy (area vs. delay) and the second one is an index.
(Default: AREA 0)
```
One of design set  `set ::env(SYNTH_STRATEGY) "DELAY 2"` its failing with routing congestion high. (MPW-4 OL tag its passed routing stage, but latest tag throwing error)
Same design works fine for "DELAY 1" failing for 0/2/3 index.
Provide solution in which scenarios we can modify these index values.
Thanks in advance!!!

**Comment by maliberty** [comment]
I don't think there is any a priori way to know what's best and that's what synthesis explorer is for.

**Comment by vijayank88** [comment, marked as answer]
`./flow.tcl -design <design-name> -synth_explore` will provide detailed chart based on AREA/DELAY.
